distribution/lib/Standard/Table/0.0.0-dev/src/Data/Type/Value_Type.enso

```diff
@@ -186,23 +186,93 @@ type Value_Type
         Value_Type.Date_Time _ -> True
         _ -> False
 
+    ## PRIVATE
+       Specifies if values of the given type can be compared for ordering.
+    has_ordering : Boolean
+    has_ordering self = case self of
+        Value_Type.Integer _ -> True
+        Value_Type.Float _ -> True
+        Value_Type.Decimal _ _ -> True
+        Value_Type.Char _ _ -> True
+        Value_Type.Date -> True
+        Value_Type.Date_Time _ -> True
+        Value_Type.Time -> True
+        ## Not all mixed types are ordered, but some can, so we allow ordering
+           for mixed columns.
+        Value_Type.Mixed -> True
+        _ -> False
+
+    ## PRIVATE
+       Specifies if values of the given type can be compared for ordering.
+    are_comparable : Value_Type -> Value_Type -> Boolean
+    are_comparable type_1 type_2 =
+        find_bucket typ = typ.if_not_error <|
+            buckets = [["Integer", "Float", "Decimal"], ["Char"], ["Date"], ["Date_Time"], ["Time"], ["Mixed"]]
+            ## TODO [RW] workaround for the bug #6258:
+            cleared_typ = Warning.clear typ
+            ctor_name = Meta.meta cleared_typ . constructor . name
+            buckets.index_of bucket->
+                bucket.contains ctor_name
+
+        bucket_1 = find_bucket type_1
+        bucket_2 = find_bucket type_2
+        if bucket_1.is_nothing || bucket_2.is_nothing then False else
+            if (type_1 == Value_Type.Mixed) || (type_2 == Value_Type.Mixed) then True else
+                bucket_1 == bucket_2
+
+
+    ## PRIVATE
+       Checks that both types can be compared for ordering and runs the
+       following action or reports an error.
+
+       The second argument may be `Nothing` if the type represents null values.
```

Comment:
Can you clarify what you mean by "represents null values"? Does this describe certain types but not others? Or is it about the possibility that `type_1` is Nothing?